packages/csharp/ArmoniK.Api.Common.Channel/ArmoniK.Api.Common.Channel.csproj

```diff
@@ -5,6 +5,8 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <PackageVersion>3.7.2</PackageVersion>
+    <PackageOutputPath>../publish</PackageOutputPath>
```

Comment:
why do you need that ?

Reply:
Only some packages were published locally on build. That enables debugging with local publishing.